client/request/handlers.go

```diff
@@ -172,6 +172,7 @@ var JSONBuilder = Handler{
 				r.Error = err
 				return
 			}
+			r.HTTPRequest.ContentLength = int64(len(raw))
```

Comment:
hmmm..... I would think would is set automatically from body. Are we not seeing it in requests?